presto-main/src/main/java/com/facebook/presto/execution/CreateViewTask.java

```diff
@@ -96,7 +96,7 @@ public String explain(CreateView statement, List<Expression> parameters)
 
     private Analysis analyzeStatement(Statement statement, Session session, Metadata metadata, AccessControl accessControl, List<Expression> parameters)
     {
-        Analyzer analyzer = new Analyzer(session, metadata, sqlParser, accessControl, Optional.empty(), parameters);
+        Analyzer analyzer = new Analyzer(session, metadata, sqlParser, accessControl, Optional.empty(), parameters, BlackholeWarningSink.getInstance());
```

Comment:
Do you have a follow up plan how to support warnings during runtime?

This would be very useful since some operators should produce warnings in runtime depending on data, e.g:
```
cast('abc   ' as varchar(3))
```
should produce
```
warning — string data, right truncation
```